src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String BUY = "buy";
+    public static final String LINE_SEPARATOR = " ";
+    public static final String SUPPLY = "supply";
+    public static final String VALUE_SEPARATOR = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String dataFromFile = readFile(fromFileName);
+        String report = createReport(dataFromFile);
+        writeToFile(report, toFileName);
+    }
+
+    public String readFile(String fileName) {
```

Comment:
let's make this method private
and the same with others

Reply:
Done